draft-yasskin-http-origin-signed-responses.md

```diff
@@ -881,14 +917,13 @@ authoritative for the PUSH_PROMISE's authority.
       ({{!RFC5280}} and other undocumented conventions). Let `path` be the path
       that was used from the `main-certificate` to a trusted root, including the
       `main-certificate` but excluding the root.
-   1. Validate that `main-certificate` includes a "status_request" extension
+   1. Validate that `main-certificate` has an `ocsp` property ({{cert-chain}})
       with a valid OCSP response whose lifetime (`nextUpdate - thisUpdate`) is
       less than 7 days ({{!RFC6960}}). Note that this does not check for
       revocation of intermediate certificates, and clients SHOULD implement
       another mechanism for that.
-   1. Validate that all certificates in `path` include
-      "signed_certificate_timestamp" extensions containing valid SCTs from
-      trusted logs. ({{!RFC6962}})
+   1. Validate that all certificates in `path` include `sct` properties
```

Comment:
Is this right, or should it only be the leaf certificate that holds an SCT?

Reply:
Under RFC 6962, only the leaf has (additional) SCTs provided by the TLS extension

Under 6962-bis, this can change, but there are no implementations of 6962-bis (nor announced) that I know of.

Reply:
I've changed it to just the leaf certificate. Thanks.